hedyorg/hedy
# [UI] Add syntax highlighting for new 'pressed' keyword (3547)

**Description**

**Adds** syntax highlighting for the new 'pressed' keyword, including some unit tests to make sure it works.

@Felienne indicated to our team that it might be nice to give the 'pressed' keyword a different color to distinguish it from other keywords. Since I'm still not that comfortable with the Hedy codebase yet, I'm still researching how to do this. I'm opening this pull request to at least get basic highlighting up and running.

Fixes #3547

**How to test**
Go to level 5 or above and enter the code snippet
```if x is pressed```
The 'pressed' keyword should now be colored!
